chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/surveillance/CreateActionPermissions.java

```diff
@@ -36,7 +36,7 @@ public boolean hasAccess() {
     public boolean hasAccess(Object obj) {
         if (!(obj instanceof Surveillance)) {
             return false;
-        } else if (getResourcePermissions().isUserRoleAdmin() || getResourcePermissions().isUserRoleOnc()) {
+        } else if (getResourcePermissions().isUserRoleAdmin()) {
```

Comment:
The way I read the ticket it looks like ROLE_ONC should continue to be able to do stuff. Is that right?